src/platform/src/main/java/org/geoserver/platform/resource/Files.java

```diff
@@ -273,6 +277,97 @@ private Files() {
         // utility class do not subclass
     }
 
+    /** @deprecated use {@link Resources#fromURL(Resource, String)} */
+    @Deprecated
+    public static File url(File baseDirectory, String url) {
```

Comment:
Confused, am I seeing double? The method below has the same signature as this one:

```
public static File url(File baseDirectory, String url) {
```

So this cannot compile. Maybe it was meant to be removed? However, this also means the PR was not built, otherwise this would have been spotted immediately